galaxy/main/signals/handlers.py

```diff
@@ -59,7 +61,7 @@ def user_logged_in_handler(request, user, **kwargs):
 
         if provider.name.lower() == 'github':
```

Comment:
I think we no longer need to iterate over Providers. We assumed GitHub above to get the social object (line #39), so might as well continue to assume GitHub here as well, and use the existing social object.